components/fxa-client/src/http_client.rs

```diff
@@ -482,7 +536,8 @@ pub fn send_authorization_request(
     let req = HawkRequestBuilder::new(Method::Post, auth_endpoint, auth_key)
         .body(serde_json::to_value(&auth_params)?)
         .build()?;
-    let resp: serde_json::Value = Client::make_request(req)?.json()?;
+    let client = Client::new();
```

Comment:
We are doing a `Client::new` in each request, is there a way of not doing that?

Reply:
Yup! I can move those functions to be inside the Client impl and have the integration tests worry about creating the client 😄